core/shared/src/main/scala/monocle/function/Each.scala

```diff
@@ -54,6 +55,16 @@ object Each extends EachFunctions with EachInstancesScalaVersionSpecific {
 
   implicit def listEach[A]: Each[List[A], A] = fromTraverse
 
+  implicit def listMapTraversal[K, V]: Traversal[ListMap[K, V], V] = new Traversal[ListMap[K, V], V] {
```

Comment:
could you inline this Traversal in the Each instance below? We only mark as implicit instances of typeclass but an optic is not a typeclass. For example, you could create a `Traversal[ListMap[K, V], V]` that only map the first and last element of the ListMap.

Reply:
Done. :-)

Reply:
cheers